types/react-native/tsconfig.json

```diff
@@ -5,6 +5,7 @@
             "es6",
             "dom"
         ],
+        "declaration": true,
```

Comment:
That's a TypeScript bug -- test cases for those should be in the TypeScript repository, not DefinitelyTyped.